tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -574,4 +587,105 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.GhQuantities",
                                     upper_bound);
   test_shift_deriv_functions_analytic(solution, grid_size, lower_bound,
                                       upper_bound);
+
+  // Check that compute items work correctly in the DataBox
+  // First, check that the names are correct
+  CHECK(GeneralizedHarmonic::Tags::PhiCompute<3, Frame::Inertial>::name() ==
+        "Phi");
+  CHECK(GeneralizedHarmonic::Tags::PiCompute<3, Frame::Inertial>::name() ==
+        "Pi");
+  CHECK(GeneralizedHarmonic::Tags::ExtrinsicCurvatureCompute<
+            3, Frame::Inertial>::name() == "ExtrinsicCurvature");
+
+  // Check that the compute items return the correct values
+  MAKE_GENERATOR(generator);
+  std::uniform_real_distribution<> distribution(-0.1, 0.1);
+
+  auto spacetime_metric =
```

Comment:
Can you use the trick here you used in #1475, where you generate and adjust the metric in a lambda, then return to a `const auto`? If so, I liked that technique and would prefer it here as well.